overlord/snapstate/handlers.go

```diff
@@ -2980,8 +2980,33 @@ func (m *SnapManager) doCheckReRefresh(t *state.Task, tomb *tomb.Tomb) error {
 }
 
 func (m *SnapManager) doAutoRefreshGate(t *state.Task, tomb *tomb.Tomb) error {
```

Comment:
we should probably follow the usual naming convention for this and call it doCondinationalAutoRefresh

Reply:
Indeed, done.